capstone/static/js/search/search-form.vue

```diff
@@ -52,14 +52,11 @@
             </div>
             <div class="col-1">
               <div class="remfield">
-                <button v-if="fields.length > 1"
-                        class="field-button active"
+                <button type="button"
+                        :class="[fields.length <= 1 ? 'disabled' : 'active', 'field-button']"
+                        :disabled="fields.length <= 1"
                         @click="removeField(field['name'])">
                 </button>
-                <button v-if="fields.length <= 1"
-                        class="field-button disabled"
-                        disabled>
```

Comment:
Doesn't this need to be either `disabled` or `disabled="disabled"` (for XHTML), or on the other hand nothing at all?

Reply:
My force-push fixed that -- in Vue, if you return `false` from a bound attribute, the attribute is excluded. `:disabled="fields.length <= 1"` will include `disabled="disabled"` for fields.length <= 1, and otherwise will include nothing.